### PR TITLE
Update MicrosoftOffice.gitignore

### PR DESCRIPTION
**Pull request by jgm1986**
**Reasons for making this change:**

 - Microsoft Visio is a tool included on Microsoft Office and it creates temporary files when you are working in a file.

**Links to documentation supporting these rule changes:** 

 - [Microsoft Visio File Format](https://msdn.microsoft.com/en-us/library/office/jj228622.aspx)

If this is a new template: 

 - [Microsoft Visio 2016 Trial](https://www.microsoft.com/en-us/evalcenter/evaluate-visio-professional-2016/)

### CODE REVIEW

**Comment by shiftkey** [comment]
@jgm1986  the example link talks about the other extensions from Visio - is `~ vsdx` the right one to ignore here?

**Comment by jgm1986** [comment]
Sorry I have not found any official place where explain the Visio  temporary files extension.

visio temporal file

I have attached an example of my own computer. As you can see, the original Visio extension file is **.vsdx** but the *autosave temporary file* contains the **.~vsdx** extension.

File: Global/MicrosoftOffice.gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@
 
 # PowerPoint temporary
 ~$*.ppt*
+
+# Visio temporary
```
Comment (shiftkey) [inline Global/MicrosoftOffice.gitignore]: Can we make this a bit more clearer? If it is related to autosave then perhaps something like this:

```
# Visio autosave temporary files
```

**Comment by shiftkey** [comment]
@jgm1986 thanks, that should be good enough for referring back to later.

Just one comment about the comment above this rule and this should be good to go.

**Comment by jgm1986** [comment]
Sure! How about this? ;)

**Comment by shiftkey** [comment]
:shipit: thanks @jgm1986!

**Pull request merged**